stl/inc/xmemory

```diff
@@ -1689,10 +1689,10 @@ _Alloc_ptr_t<_Alloc> _Uninitialized_move(
     // clang-format on
 
     using _Ptrval = typename _Alloc::value_type*;
-    _Seek_wrapped(
-        _Dest, _Uninitialized_move_al_unchecked(_UFirst, _ULast, _UDest, _Al,
-                   bool_constant < _Ptr_move_cat<decltype(_UFirst), _Ptrval>::_Really_trivial
-                       && _Uses_default_construct<_Alloc, _Ptrval, decltype(_STD move(*_UFirst))>::value > {}));
+    _Seek_wrapped(_Dest,
+        _Uninitialized_move_al_unchecked(_UFirst, _ULast, _UDest, _Al,
+            bool_constant < _Can_memmove_uninitialized<decltype(_UFirst),
+                    _Ptrval> && _Uses_default_construct<_Alloc, _Ptrval, decltype(_STD move(*_UFirst))>::value > {}));

```

Comment:
It seems like the spacing around `bool_constant` is different here than in other occurrences (specifically before and after the `<` and `>`), and it also looks like it is preexisting in the code before changes.  Is there a reason for this?

Reply:
I believe this is clang format getting confused by the `_Uses_default_construct<_Alloc, _Ptrval, decltype(_STD move(*_UFirst))>::value > {}` afterwards

I was not able to create a simple reproducer though